helm/event-exporter-app/templates/vpa.yaml

```diff
@@ -1,23 +1,20 @@
-{{ if .Capabilities.APIVersions.Has "autoscaling.k8s.io/v1" }}
```

Comment:
Why remove this check?

Reply:
But doesn't that then cause issues with dependencies on creating a new cluster? I recall something like that but not the specifics.

Reply:
Hehe, fun fact: If you add something like this, you will end up with an app not having VPA at all and still being `deployed` successfully. Also not good. With this requirement, the chart operator / Flux Helm controller is just retrying installation until it works.